chore: improvements to l10n page (#40355)
mdn/content: files/en-us/mdn/community/translated_content/index.md

sidebar: mdnsidebar
---

MDN Web Docs contains close to 14000 pages of documentation in the `en-US` locale.

> The English documentation is available in eight locales maintained by a dedicated and diverse community of contributors.

Author: ```suggestion
MDN content is translated into eight locales maintained by a dedicated and diverse community of contributors.
```

> The English documentation is available in eight locales maintained by a dedicated and diverse community of contributors.

Author: Thanks, Chris - want to have another look?
